addons/photo_collage/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Photo Collage",
+    'summary': """Controller for the photo collage""",
+    'description': """
+        Controller for the photo collage
```

Comment:
It's a full module with its own forms, views etc., so I wouldn't describe it as just a "controller".